src/AppInstallerCLICore/TableOutput.h

```diff
@@ -30,6 +30,7 @@ namespace AppInstaller::CLI::Execution
     }
 
     // Enables output data in a table format.
+    // TODO: Improve for use with sparse data.

```

Comment:
>TODO: Improve for use with sparse data. [](start = 7, length = 39)

what kind of improvement are you considering?

Reply:
Probably a bit more control over minimum/maximum/collapse state for columns.  For instance, list could easily have only 1 update to show but far to the end and the code here would not show it.  I suppose that could be fixed by simply sorting the updates to the beginning of the output though.

---
In reply to: [506790702](https://github.com/microsoft/winget-cli/pull/598#discussion_r506790702) [](ancestors = 506790702)